charts/compute/templates/certificate.yaml

```diff
@@ -13,5 +13,5 @@ spec:
     algorithm: RSA
     encoding: PKCS8
     size: 4096
-  secretName: unikorn-compute-client-certificate
+  secretName: {{ .Release.Namespace }}-client-certificate
```

Comment:
Note: this fixes a bug where the application has to be installed in a specific namespace.